src/ai/susi/server/api/cms/SkillMetricsDataService.java

```diff
@@ -188,6 +188,34 @@ public int compare(JSONObject a, JSONObject b) {
         JSONArray ratingsData = getSlicedArray(jsonValues, count);
         skillMetrics.put("rating", ratingsData);
 
+        // Get games skills based on ratings
+        Collections.sort(jsonValues, new Comparator<JSONObject>() {
+            @Override
+            public int compare(JSONObject a, JSONObject b) {
+                float valA;
+                float valB;
+                int result=0;
+
+                if (!a.get("group").toString().equals("Games, Trivia and Accessories")){
+                    return 1;
+                }
+                if (!b.get("group").toString().equals("Games, Trivia and Accessories")){
+                    return -1;
+                }
+                try {
+                    valA = a.getJSONObject("skill_rating").getJSONObject("stars").getFloat("avg_star");
+                    valB = b.getJSONObject("skill_rating").getJSONObject("stars").getFloat("avg_star");
+                    result = Float.compare(valB, valA);
+                } catch (JSONException e) {
+                    e.printStackTrace();
+                }
+                return result;
+            }
+        });
+
+        JSONArray topGamesData = getSlicedArray(jsonValues, count);
+        skillMetrics.put("topGames", topGamesData);
```

Comment:
In the context of having more flexibility with the preferred group, the name of the metrics "topGames" does not make sense. Please rename "topGames" to "top".